store/src/java/com/zimbra/cs/imap/ImapHandler.java

```diff
@@ -515,6 +520,11 @@ boolean executeRequest(ImapRequest req) throws IOException, ImapException {
                     }
                     checkEOF(tag, req);
                     return isProxied ? imapProxy.proxy(req) : doFETCH(tag, sequence, attributes, parts, byUID, modseq);
+                } else if (command.equals("FLUSHCACHE")) {
```

Comment:
seems unlikely that IMAP will be extended with 'FLUSHCACHE' or 'RELOADLC' commands, but to avoid any potential namespace conflict, could you rename these commands to something like X-ZIMBRA-FLUSHCACHE and X-ZIMBRA-RELOADLC?